device/celestica/x86_64-cel_silverstone-r0/Silverstone-128x100/port_config.ini

```diff
@@ -0,0 +1,129 @@
+# name          lanes       alias           index       speed
+Ethernet0       33,34       QSFP1/1         1           100000
+Ethernet1       35,36       QSFP1/2         1           100000
```

Comment:
This should be Ethernet2 instead of Ethernet1.  We should reserve one interface per lane.

Reply:
I agree with you, I will update it.

Reply:
updated in https://github.com/Azure/sonic-buildimage/pull/5424/commits/2d79cb570370808e69015139681fe907667c81a8